proposals/0309-unlock-existential-types-for-all-protocols.md

```diff
@@ -146,7 +146,7 @@ func test(_ collection: RandomAccessCollection) {
 ```
 ___
 
-This way, a protocol or protocol extension member (method/property/subscript/initializer) may be used on an existential value iff the following criteria hold:
+This way, a protocol or protocol extension member (method/property/subscript/initializer) may be used on an existential value if the following criteria hold:
```

Comment:
`iff` is short for "if and only if", which is likely intentional.

Reply:
TIL - I've never seen it shortened like that before (having studied math in another language). Would you agree that it'd be clearer, if it's spelled out fully?

Reply:
"iff" was removed from public API documentation: apple/swift#27115

Reply:
Updated to use the spelled out version. From the context it seems intentional and not a typo.

Reply:
Thank you!